ansible/playbooks/service/pdns.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Copyright (C) 2020 CipherMail B.V. <https://www.ciphermail.com/>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Manage PowerDNS authoritative server
+  hosts: [ 'debops_service_pdns' ]
+  become: True
+
+  environment: '{{ inventory__environment | d({})
+                   | combine(inventory__group_environment | d({}))
+                   | combine(inventory__host_environment  | d({})) }}'
+
+  roles:
+
+    - role: etc_services
+      tags: [ 'role::etc_services', 'skip::etc_services' ]
+      etc_services__dependent_list:
+        - '{{ pdns__etc_services__dependent_list }}'
+      when: pdns__api or pdns__metrics
+
+    - role: ferm
+      tags: [ 'role::ferm', 'skip::ferm' ]
+      ferm__dependent_rules:
+        - '{{ pdns__ferm__dependent_rules }}'
```

Comment:
For example nginx dependent_rules are missing here.

Reply:
Right, I never thought about this. I see that it's done like this with `debops.owncloud` as well. The downside of this approach is that it basically makes copies of the `service/nginx` playbook. Wouldn't it be cleaner to just instruct the user to add the host to the `[debops_service_nginx]` inventory group as well? Just to prevent code duplication and to keep some separation between the roles?

Reply:
This is a tradeoff between maintenance overhead and runtime. I guess we favor runtime but you are right. I am not fully happy with the duplication either.

Reply:
Addressed in e938bf0f5c36d5c4fa8f21f41f7beec0bfeb9a6f